app/forms/declaration/DeclarationAdditionalActors.scala

```diff
@@ -45,7 +45,7 @@ object DeclarationAdditionalActors extends DeclarationPage {
   val eoriPattern = "[0-9a-zA-Z]{1,17}"
 
   val mapping = Forms.mapping(
-    "eori" -> optional(text().verifying("supplementary.eori.error", _.matches(eoriPattern))),
+    "eori" -> optional(text().verifying("supplementary.eori.error.format", _.matches(eoriPattern))),
```

Comment:
Here the validation is wrong. As you can see we're validating this with old eori pattern `[0-9a-zA-Z]{1,17}` Could you use here `isValidEORI`?

Reply:
Done

Reply:
Done